### DISCUSSION TITLE
Edit Plex Config

### DISCUSSION THREAD

**Discussion by Nerdflex** [Luctia/ezarr#56] [Help wanted]
Hey Luctia! 

Really appreciate the time and effort to put this all together. I definitely got everything I wanted set up pretty easily. So thanks! 

I was wondering if you could help me out, I am having trouble locating the Plex Config file ( ?) so I can add 
`--device=/dev/dri:/dev/dri`
For hardware acceleration. 

I am not sure where to locate it. I attempt to find it in the container for plex as well as the plex-config folder that is created and no dice. 

am I missing it? Where is it normally located? and How would I access it via SSH ( headless server ) 

Thanks again for your time and assistance!!

**Comment by Luctia** [comment]
Hi, what exactly do you mean by plex config file?
